gradle/libs.versions.toml

```diff
@@ -263,7 +263,7 @@ jsonassert = "org.skyscreamer:jsonassert:1.5.0"
 junit = "junit:junit:4.13.2"
 junit-ext = "androidx.test.ext:junit:1.1.3"
 leakcanary = "com.squareup.leakcanary:leakcanary-android:2.10"
-material = "com.google.android.material:material:1.5.0"
+material = "com.google.android.material:material:1.9.0"
```

Comment:
I updated material lib to a stable version 1.9.0  for targetSdkVerrsion 33. This was primarily to [prevent private resource warning for colorPrimaryContainer](https://rb.gy/m8okwy).

If you think it is too big bump for this PR, I can update it to 1.6.0 which includes a fix.

Reply:
Thanks for flagging this. I think it's fine. 👍